### DISCUSSION TITLE
Error while creating Release

### DISCUSSION THREAD

**Discussion by chennamjaya** [learndevopsonline/learndevopsonline#1692] [Q&A]
I am trying to run tail -f /opt/userdata.log in the backend server and i am getting the following error. Same with the frontend as well.
![Screenshot 2023-12-21 164931](https://github.com/learndevopsonline/learndevopsonline/assets/62970385/95d5d21f-3b2b-48eb-a5e3-9a808c470b08)

**Comment by sreedharm07** [comment]
your password is dontnow  or dontknow ?

**Reply by chennamjaya** [reply]
I have rectified it, its resolved now

**Reply by chennamjaya** [reply]
Was able to run the backend successfully but getting error while running the frontend servers: 
![Screenshot 2023-12-22 104708](https://github.com/learndevopsonline/learndevopsonline/assets/62970385/3612b4d2-5898-4fd1-9b4f-e112258f5cb8)

**Reply by sreedharm07** [reply]
completed the issue in session
